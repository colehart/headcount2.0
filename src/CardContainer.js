import React from 'react'

const CardContainer = () => {
  return(
    <main className="CardContainer">
    </main>
  )
}

export default CardContainer