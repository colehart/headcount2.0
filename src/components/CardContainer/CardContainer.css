/*CARDCONTAINER COMPONENT RELATED CSS GOES HERE*/
.CardContainer {
  grid-row: 2 / 3;
  grid-column: 1 / 2;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
}