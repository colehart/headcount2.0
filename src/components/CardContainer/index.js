import React from 'react'
import PropTypes from 'prop-types'
import Card from '../Card'
import './CardContainer.css'

const CardContainer = ({districts}) => {

  const cards = Object.keys(districts).map((district, index) => {
    console.log(`district: ${district}`)
    return <Card key={index} location={district} {...districts[district]} />
  })

  return(
    <main className="CardContainer">
       { cards }
    </main>
  )
}

CardContainer.propTypes = {
  districts: PropTypes.object.isRequired
}

export default CardContainer