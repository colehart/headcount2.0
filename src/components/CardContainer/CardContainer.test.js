import React from 'react'
import { shallow } from 'enzyme'
import CardContainer from './index'

describe('CardContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardContainer />)
  })

  it('should have cards', () => {
    expect(wrapper).toMatchSnapshot()
  })
})