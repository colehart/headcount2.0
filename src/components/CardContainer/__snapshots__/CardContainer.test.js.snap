// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer should have cards 1`] = `
<main
  className="CardContainer"
/>
`;
